### ISSUE TITLE
Build failing after updating codec scripts to upstream in my fork

### ISSUE THREAD

**Issue by rokibhasansagar** [BtbN/FFmpeg-Builds#208]
In my slightly-customized fork at https://github.com/rokibhasansagar/FFmpeg-Builds, after updating the codecs using `util/update_scripts.sh`, ffmpeg build fails after `configure` command in `build_ffmpeg` job, saying `ERROR: x265 not found using pkg-config`

Link to Failed Workflow Run: https://github.com/rokibhasansagar/FFmpeg-Builds/actions/runs/3336537955

Could you please help me figure out what might be the problem with `pkg-config` in my fork?

Additionally, could you also try updating the codecs in your repo and see if the same thing happens or not?

**Comment by BtbN** [comment]
check the config.log for the specific error (the ffbuild directory is left in place on a failed build), and then hope it's nothing too crazy that can be just fixed.
I'll do another round of dependency updates this weekend most likely and will probably then see what's going on.

**Comment by rokibhasansagar** [comment]
To find out the cause, I have followed these steps so far:

1. Ran a Remote SSH Session in GH Actions
2. Cloned the repo
3. I chose `linux64-nonfree-5.1` for test, Pulled the docker image (`ghcr.io/rokibhasansagar/ffmpeg-builds/linux64-nonfree-5.1:latest`) which is needed for `build_ffmpeg` job for specific `{TARGET}-{VARIANT}`
4. Ran `./build.sh linux64 nonfree 5.1`

Got same result as Workflow Run at Github.

I extracted the full `config.log` file for the specific variant: [config.tar.gz](https://github.com/BtbN/FFmpeg-Builds/files/9882485/config.tar.gz)

Only the part of Log output for x265: [config.x265-part.tar.gz](https://github.com/BtbN/FFmpeg-Builds/files/9882588/config.x265-part.tar.gz)

I hope it may give you some insight for the problem.
